module-4/backend/week_four.md

```diff
@@ -4,8 +4,48 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-TBD  
+1. What's your favorite project management tool?
+
+I like both waffle and pivotal, pros and cons with both. Need more reps to have a matured opinion
+
+2. Why do we create staging environments?
+
+Mimic production
```

Comment:
Be able to talk about what problems you can run into between development and production.